docs/docs/40-operator-guide/40-security/30-access-controls.md

```diff
@@ -110,31 +123,41 @@ As previously mentioned, _most_ access controls are managed at the project level
 by project admins, however, there are two ways in which an operator can also
 map users to `ServiceAccount` resources.
 
-#### `api.oidc.admins` / `api.oidc.viewers`
+#### Built-in System Roles
+
+The `api.oidc.admins`, `api.oidc.projectCreators`, `api.oidc.users`, and
+`api.oidc.viewers` configuration options of the Kargo Helm chart permit an 
+operator to map users with specific claims to _system-wide_ admin, 
+project creator, user, and viewer roles respectively. 
 
-The `api.oidc.admins` and `api.oidc.viewers` configuration options of the Kargo
-Helm chart permit an operator to map users with specific claims to
-_system-wide_ admin and viewer roles respectively. If, for example, every user
-in the group `devops` should be an admin, and every user in the group
-`developers` should be a viewer, you would set these accordingly:
+If, for example:
+
+- every user in the group `devops` should be an admin
+- every user in the group `sales` should be a project creator
+- every user in the group `developers` should be user
+- and every user in the group `support` should be a viewer
+
+You would use the following configuration:
 
 ```yaml
 api:
   oidc:
     # ... omitted for brevity ...
     admins:
-      claims:
-        groups:
-        - devops
+      claims: '{"groups":["devops"]}'
+    projectCreators:
+      claims: '{"groups":["sales"]}'
```

Comment:
This is an odd example -- it would be odd for sales to touch Kargo at all. Maybe change this to something like "team-leads"?

```suggestion
    projectCreators:
      claims: '{"groups":["sales"]}'
```